clicommand/pipeline_upload.go

```diff
@@ -43,7 +43,9 @@ Description:
    - buildkite/pipeline.json
 
    You can also pipe build pipelines to the command allowing you to create
-   scripts that generate dynamic pipelines.
+   scripts that generate dynamic pipelines.The configuration file has a
```

Comment:
Missing a space. You could also consider starting a new paragraph for this.

```suggestion
   scripts that generate dynamic pipelines. The configuration file has a
```